hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/VertexAPI.java

```diff
@@ -259,22 +314,50 @@ private static void checkBatchSize(HugeConfig config,
         int max = config.get(ServerOptions.MAX_VERTICES_PER_BATCH);
         if (vertices.size() > max) {
             throw new IllegalArgumentException(String.format(
-                      "Too many counts of vertices for one time post, " +
+                      "Too many vertices for one time post, " +
                       "the maximum number is '%s'", max));
         }
     }
 
-    @JsonIgnoreProperties(value = {"type"})
-    private static class JsonVertex implements Checkable {
+    //TODO: Should support AUTOMATIC Id strategy ?
+    private Object getVertexId(HugeGraph g, String labelId, JsonVertex vertex) {
```

Comment:
If let getVertexId() return Id then it can be more simple and unified in the future.